libs/hdf-converters/test/mappers/forward/trufflehog_mapper.spec.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs';
+import {TrufflehogMapper} from '../../../src/trufflehog-mapper';
+import {omitVersions} from '../../utils';
+
+describe('trufflehog_mapper', () => {
+  it('Successfully converts trufflehog targeted at a local/cloned repository data', () => {
+    const mapper = new TrufflehogMapper(
+      fs.readFileSync(
+        'sample_jsons/trufflehog_mapper/sample_input_report/trufflehog.json',
+        {encoding: 'utf-8'}
+      )
+    );
+
+     fs.writeFileSync(
+       'sample_jsons/trufflehog_mapper/trufflehog-hdf.json',
+       JSON.stringify(mapper.toHdf(), null, 2)
+    );
```

Comment:
```suggestion
     // fs.writeFileSync(
     //   'sample_jsons/trufflehog_mapper/trufflehog-hdf.json',
     //   JSON.stringify(mapper.toHdf(), null, 2)
     // );
```
Ensure that we aren't pushing up unexpected file writes to the repo.